flutter/lib/screens/qr_code_screen.dart

```diff
@@ -0,0 +1,138 @@
+//Flutter packages
+import 'dart:async';
+import 'package:flutter/material.dart';
+//Third party packages
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:lottie/lottie.dart';
+import 'package:provider/provider.dart';
+import 'package:qr_flutter/qr_flutter.dart';
+import 'package:simple_connection_checker/simple_connection_checker.dart';
+//My Packages
+import '../providers/superformula_provider.dart';
+import 'offline_screen.dart';
+
+class QRCCodeScreen extends StatefulWidget {
+
+  static const routeName = "/qr_code_screen";
+  const QRCCodeScreen({Key? key}) : super(key: key);
+
+  @override
+  _QRCCodeScreenState createState() => _QRCCodeScreenState();
+}
+
+class _QRCCodeScreenState extends State<QRCCodeScreen> {
+
+  //Object declarations
+
+  late Timer _timer;
+  int _time = 0;
+  String _timeInString = "";
+
+  //Offline handler
+  StreamSubscription? _offlineStreamSubscription;
+
+  @override
+  void initState() {
+    super.initState();
+
+    //Simple connection checker object init
+    SimpleConnectionChecker _simpleConnectionChecker = SimpleConnectionChecker()
```

Comment:
**suggestion:** same thing here, it'd significantly dry up the code and logic spread throughout